src/widgets.js

```diff
@@ -197,6 +197,38 @@ define([
         return widget;
     };
 
+    var multilineText = function (mug, options) {
+        var widget = normal(mug, options);
+
+        widget.input = $("<textarea></textarea>")
```

Comment:
Too bad `normal` is hard-coded to create it's own `<input />` DOM element, and which this is overwriting. Wonder if it would make sense to add `options.input`, that, if present, would be assigned to `widget.input` by `normal`?